src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerTest.java

```diff
@@ -130,35 +118,10 @@ void setUp() {
         );
     }
 
-    @Test
-    void should_return_a_500_when_id_invalid() throws Exception {
```

Comment:
can we please check invalid id check is covered in the R2 code? I.e. against CFTTaskDb